batch/create/create_batch_notifications.js

```diff
@@ -16,28 +16,38 @@
 
 'use strict';
 
-async function main() {
-  // [START batch_notifications]
-  // Imports the Batch library
-  const batchLib = require('@google-cloud/batch');
-  const batch = batchLib.protos.google.cloud.batch.v1;
+// [START batch_notifications]
+// Imports the Batch library
+const batchLib = require('@google-cloud/batch');
+const batch = batchLib.protos.google.cloud.batch.v1;
+
+// Instantiates a client
+const batchClient = new batchLib.v1.BatchServiceClient();
 
-  // Instantiates a client
-  const batchClient = new batchLib.v1.BatchServiceClient();
+/**
+ * TODO(developer): Update these variables before running the sample.
+ */
+// Project ID or project number of the Google Cloud project you want to use.
+const PROJECT_ID = 'project-id';
+// Name of the region you want to use to run the job. Regions that are
+// available for Batch are listed on: https://cloud.google.com/batch/docs/get-started#locations
+const REGION = 'europe-central2';
+// The name of the job that will be created.
+// It needs to be unique for each project and region pair.
+const JOB_NAME = 'job-name-batch-notifications';
+// The Pub/Sub topic ID to send the notifications to.
+const TOPIC_ID = 'topic-id';
 
-  /**
-   * TODO(developer): Update these variables before running the sample.
-   */
-  const PROJECT_ID = await batchClient.getProjectId();
-  // Name of the region you want to use to run the job. Regions that are
-  // available for Batch are listed on: https://cloud.google.com/batch/docs/get-started#locations
-  const REGION = 'europe-central2';
-  // The name of the job that will be created.
-  // It needs to be unique for each project and region pair.
-  const JOB_NAME = 'job-name-batch-notifications';
-  // The Pub/Sub topic ID to send the notifications to.
-  const TOPIC_ID = 'topic-id';
+async function main() {
+  await callCreateBatchNotifications(PROJECT_ID, REGION, JOB_NAME, TOPIC_ID);
```

Comment:
I'm not really on board with this approach. We end up with a pointless main function to call another function. This could just as easily be done in line 118. But it's also a very different approach from your previous PR (#3778), which I prefer over to initialization happening at the top level of a module. More importantly, however, is that we need to be consistent.

Reply:
the reason for these changes was to have a ability to manipulate values in tests. It is difficult to write tests, when we use values like `project-id` or `topic-id` in the implementation, because they do not really exist. but I see your point and will revert these changes :)